lib/rspec/rails/view_rendering.rb

```diff
@@ -44,6 +44,13 @@ def render_views?
       #
       # @private
       class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+
+        def initialize(path, pattern=nil)
+          unless path.is_a?(::ActionView::Resolver)
+            super
+          end
+        end
+
```

Comment:
We don't want a custom resolver in the path, the aim is to disable view rendering, my understanding is that would do the opposite.